yarn.lock

```diff
@@ -7466,6 +7466,11 @@ prelude-ls@~1.1.2:
   resolved "https://registry.yarnpkg.com/prelude-ls/-/prelude-ls-1.1.2.tgz#21932a549f5e52ffd9a827f570e04be62a97da54"
   integrity sha512-ESF23V4SKG6lVSGZgYNpbsiaAkdab6ZgOxe52p7+Kid3W3u3bxR4Vfd/o21dmN7jSt0IwgZ4v5MUd26FEtXE9w==
 
+prettier@2.7.1:
```

Comment:
Generally speaking, all lock files should not be committed (package.lock.json, yarn.lock) as this prevents semver-safe upgrades to libraries and can cause cross-platform build issues.